getting-started/trino-config/catalog/iceberg.properties

```diff
@@ -0,0 +1,6 @@
+connector.name=iceberg
```

Comment:
does the "Copyright" notice need to be included in every file?

Reply:
yes, basically all

Reply:
Added!